spec/controllers/your_details_controller_spec.rb

```diff
@@ -100,6 +102,63 @@ module Wpcc
       end
     end
 
+    describe 'sessions' do
+      after { Timecop.return }
+
+      it "'wpcc_expires_at' key should always be updated" do
+        session[:wpcc_expires_at] = 5.minutes.ago
+        get :new, nil, some_session
+
+        expect(session[:wpcc_expires_at]).to be > 5.minutes.ago
+      end
+
+      context 'when session has expired' do
+        before do
+          wpcc_expires_at = 5.minutes.ago
```

Comment:
do we need to set this as a variable if it's obvious once you get to the key below what it is?